src/testcase.lisp

```diff
@@ -35,7 +35,7 @@
 (defun read-from-stream (stream)
   (let ((eof nil)
         (results '())
-        (header-line (read-line stream nil nil))
+        (header-line (string-right-trim '(#\Return) (read-line stream nil nil)))
```

Comment:
This could be wrong, because string-trim on `nil` returns "something" that is not `nil`. So unless the test file is empty, this should be working.